src/makeCalculator.js

```diff
@@ -4,7 +4,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(number) {
+      this.result += number;
+    },
+    subtract(number) {
+      this.result -= number;
+    },
+    multiply(number) {
+      this.result *= number;
+    },
+    divide(number) {
+      if (number !== 0) {
+        this.result /= number;
+      } else {
+        Error('Cannot divide by zero');
```

Comment:
The `Error` function should be used with `throw` to actually raise an error. Currently, it just creates an Error object without throwing it.